components/navbar.tsx

```diff
@@ -20,6 +20,7 @@ import WarningModal from "./warningModal";
 const navigation = [
   { name: "Playground", href: "/" },
   { name: "Actions", href: "/actions" },
+  { name: "LLM", href: "/llm" },
```

Comment:
I wonder whether `LLM` is a bit jargony for developers not involved in this kind of thing.

Maybe `Model` or `AI` ?